cmd/workonCmd.go

```diff
@@ -49,11 +49,21 @@ var workonCmd = &cobra.Command{
 			}
 			namespaces, err := clientset.CoreV1().Namespaces().List(metav1.ListOptions{})
 			if err != nil {
```

Comment:
is it possible to log the error without breaking the picker - might help reduce confusion if this issue is encountered.

Reply:
Good idea. done that.